pytorch_pfn_extras/onnx/export_testcase.py

```diff
@@ -279,10 +279,23 @@ def export_testcase(
     os.makedirs(out_dir, exist_ok=True)
     if isinstance(args, torch.Tensor):
         args = args,
-    input_names = kwargs.pop(
-        'input_names',
-        ['input_{}'.format(i) for i in range(len(args))])
-    assert len(input_names) == len(args)
+
+    # We unroll list args and generate names for each tensor.
+    gen_input_names = []
+    unrolled_args = []
+
+    def append_input_name(prefix: str, arg: Any) -> None:
```

Comment:
How about supporting also `dict` and namedtuple types like `_normalize_outputs` function in _logic.py?

https://github.com/pfnet/pytorch-pfn-extras/blob/c40cd025814bc394a58c4d6634ed04eab8e3a589/pytorch_pfn_extras/handler/_logic.py#L27-L38